.github/workflows/test_build_backend_image.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Test building backend image
+
+on:
+  pull_request:
```

Comment:
Warum nicht `on:push:`?

Reply:
weil das branches angibt für die das laufen soll, während pull_request sagt, welche ziel-branches der PR haben soll. Man muss dafür etwas tiefer in die Doku einsteigen: https://stackoverflow.com/a/65096459 ist die Erklärung